src/commands.rs

```diff
@@ -167,6 +179,12 @@ enum ConfigSubcommand {
         #[arg(value_parser=NonEmptyStringValueParser::new())]
         name: Option<String>,
         // TODO: Support --show-origin once mehcode/config-rs#319 is done.
```

Comment:
FYI, we shouldn't need to wait for this bug to fixed now that we have the `LayeredConfigs` type.

Reply:
Oh cool. Updated to TODO for now.